_javascripts/src/lib/search.js

```diff
@@ -0,0 +1,56 @@
+import sidebarNavigation from './sidebar_navigation'
+
+export default function () {
+  var searchBox = document.querySelector("input[type='search']")
```

Comment:
We're using Babel. Let's use `const` and `let`. Convention is always `const` unless you need to reassign.